### DISCUSSION TITLE
What is init.bat actually performing

### DISCUSSION THREAD

**Discussion by abdulbadii** [cmderdev/cmder#2986] [Q&A]
One kind enough shed a light all about what actually init.bat of;
`
C:\cmder\vendor\init.bat
`
is affecting or performing for its important points?
(as think it is so while need to reduce the generated paths in PATH)

**Comment by daxgames** [comment]
1. It figures out where Cmder was extracted.
2. Adds Cmder to the path.
3. Locates the newest git install on the system and adds it to the path if not already there.
4. Creates the `%cmder_root%\config` directory and files.
5. Runs the user config files in the `%cmder_root%\config` folder to setup the environment and aliases.

**Comment by abdulbadii** [comment]
At 4 or 5:
`Runs the user config files in the %cmder_root%\config folder to setup the environment and aliases.`

What more clearly are the aliases and environment variables, else than PATH (as it's known easily by everyone by `echo %PATH%`)
